plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java

```diff
@@ -54,6 +56,129 @@ public void destroy()
         TestingPhoenixServer.shutDown();
     }
 
+    @Override
+    protected boolean supportsDelete()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsViews()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsArrays()
+    {
+        return true;
+    }
+
+    @Override
+    protected boolean supportsCommentOnTable()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsCommentOnColumn()
+    {
+        return false;
+    }
+
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        throw new SkipException("Phoenix connector does not support column default values");
+    }
+
+    @Override
+    public void testRenameTable()
+    {
+        assertThatThrownBy(super::testRenameTable)
+                // TODO (https://github.com/trinodb/trino/issues/7204) support table rename in Phoenix
+                .hasMessageContaining("Syntax error. Encountered \"RENAME\"");
+    }
```

Comment:
`throw new SkipException`, the test should not show up as passing

(same below)

Reply:
Thanks. https://github.com/trinodb/trino/pull/7214